bot.js

```diff
@@ -35,6 +35,19 @@ const forTaco = controller => {
             } tacos remaning.. And you tried to give *${tacosGiven}*`
           );
         }
+      } else if (userIndex === senderIndex) {
+        bot.api.reactions.add(
+          {
+            timestamp: message.ts,
+            channel: message.channel,
+            name: "fu"
```

Comment:
can we make it usable by every slack ? (so use `middle_finger`)

but I think using another emoji is more in the "taco spirit " (like no insult etc...)


What about 😏 ?